tests/e2e/vmlistener_test.go

```diff
@@ -16,6 +16,8 @@
 // In case of vm/esx host restart vDVS cleans up stale attachment details
 // the volume.
 
+// +build unstable
```

Comment:
Fine. Before we figure out how to do this, we can use some workaround - for example, add a "disable" prefix to the test case name to skip one single unstable test case.